08-transport.md

```diff
@@ -408,11 +399,11 @@ construction, and `16 bytes` for a final authenticating tag.
 **Sender Actions:**
 
 
-  * `c = encryptWithAD(temp_k, n, h, s.pub.serializeCompressed())`
+  * `c = encryptWithAD(temp_k, 2, h, s.pub.serializeCompressed())`
     * where `s` is the static public key of the initiator.
 
 
-  * `h = SHA-256(h || c)`
```

Comment:
The prior version (as written currently) is correct. The ciphertext gets mixed in, instead of the raw public key. The handshake digest is meant to authenticate everything that's sent over the wire. In this case the encrypted pubkey, not the pubkey itself is being sent.